trino/upload_new_trino_version.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+VERSION=${1:?"Missing version number argument (arg 1)"}
+NEXUS_USER=${2:?"Missing Nexus username argument (arg 2)"}
+
+read -r -s -p "Nexus Password: " NEXUS_PASSWORD
+echo ""
+
+# https://stackoverflow.com/questions/4632028/how-to-create-a-temporary-directory
+# Find the directory name of the script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# the temp directory used, within $DIR
+WORK_DIR=$(mktemp -d -p "$DIR")
+
+# check if tmp dir was created
+if [[ ! "$WORK_DIR" || ! -d "$WORK_DIR" ]]; then
+  echo "Could not create temp dir"
+  exit 1
+fi
+
+# deletes the temp directory
+function cleanup {
+  rm -rf "$WORK_DIR"
+}
+
+# register the cleanup function to be called on the EXIT signal
+trap cleanup EXIT
+
+cd "$WORK_DIR" || exit
+
+bin_file=trino-server-$VERSION.tar.gz
+
+echo "Downloading Trino (this can take a while, it is intentionally downloading from a slow mirror that contains all old versions)"
+curl --fail -LOs "https://repo1.maven.org/maven2/io/trino/trino-server/${VERSION}/${bin_file}"
+curl --fail -LOs "https://repo1.maven.org/maven2/io/trino/trino-server/${VERSION}/${bin_file}.asc"
+curl --fail -LOs "https://repo1.maven.org/maven2/io/trino/trino-server/${VERSION}/${bin_file}.sha1"
+
+# It is probably redundant to check both the checksum and the signature but it's cheap and why not
+echo "Validating SHA1 Checksum"
+if ! (sha1sum "${bin_file}" | cut -d " " -f 1 | diff -Z - "${bin_file}.sha1"); then
+  echo "ERROR: The SHA1 sum does not match"
+  exit 1
+fi
+
+echo "Validating signature"
+echo '--> NOTE: Make sure you have added the public key (https://app.nuclino.com/Stackable/Knowledge-Base/Finding-GPG-keys-for-Maven-packages-6b7b0324-8f0f-4b0f-a6e8-aa97cea5512c) to GPG: https://www.apache.org/info/verification.html'
```

Comment:
_Which_ PGP key? GPG also doesn't seem to like grabbing it...

```
n@emeritus:~/Downloads$ gpg --receive-keys C328250FE23A2420814521EC0EB69F76FD171538
gpg: key 0EB69F76FD171538: no user ID
gpg: Total number processed: 1
n@emeritus:~/Downloads$ echo $?
2
```

Reply:
Without a solid web of trust or some form of key pinning then GPG signatures are worth no more than a plain old SHA2 signature.

Reply:
Yeah im for removing this, @Jimvin likes to have it in. I dont see the point currently.

Reply:
(`--no-default-keyring` is required to force it to import the key even if it already exists in your default keyring.. gpgv ignores the default keyring so no issue there)

Reply:
This should hopefully also remove the manual intervention and Nuclino dependency..

Reply:
https://github.com/stackabletech/docker-images/pull/487/commits/21230a6a25575baff6d634ca64fbce1ff949532e